e2e/test/iothub/messaging/MessageReceiveE2EPoolAmqpTests.cs

```diff
@@ -20,56 +19,10 @@ public class MessageReceiveE2EPoolAmqpTests : E2EMsTestBase
     {
         private readonly string DevicePrefix = $"E2E_{nameof(MessageReceiveE2EPoolAmqpTests)}_";
 
-        [Ignore]
-        [LoggedTestMethod]
-        public async Task Message_DeviceSak_DeviceReceiveSingleMessage_SingleConnection_Amqp()
```

Comment:
Removing tests already covered in single device tests that shouldn't be interesting for AMQP pooling.